projects/cds/src/profiletag/services/spartacus-event.service.ts

```
@@ -50,4 +64,22 @@ export class SpartacusEventService {
       }))
     );
   }
+
+  /**
+   * Listens to successful logins and pushes the event for profiletag to pick it up further
+   */
+  loginSuccessful(): Observable<boolean> {
+    return this.actionsSubject.pipe(
+      filter(
+        action =>
+          action.type === AuthActions.LOGIN ||
+          action.type === AuthActions.LOGOUT
+      ),
+      distinctUntilChanged(
```

You could `map` to the action to `action.type`. Then you can have just `distinctUntilChanged()`.

---

Removed the `distinctUntilChanged` after clarifying that a login twice in a row would be a serious bug and it is better to fail fast